inspector/src/components/actionTabs/tabs/propertyGrids/animations/popupComponent.tsx

```diff
@@ -14,6 +14,33 @@ export class PopupComponent extends React.Component<IPopupComponentProps, { isCo
     private _container: HTMLDivElement;
     private _window: Window | null;
 
+    private _CopyStyles(sourceDoc: HTMLDocument, targetDoc: HTMLDocument) {

```

Comment:
We should reuse the code used in inspector.ts (and also remove it from popup.ts)

Reply:
ok. let me further explore how the code in inspector.ts works for popups.

Reply:
Sorry I meant: just use one single _CopyStyles across the Inspector codebase ;)

Reply:
Let me know what you think about using the Inspector class like in the last commit :)

Reply:
All good! Can you update your popup class in the next PR to use the same? Thanks a ton

Reply:
Sure I can. I can create a separate PR for that. Just to make sure, you mean the one in node editor/src/sharedComponents/popup.ts? :)

Reply:
Actually I meant this one: https://github.com/BabylonJS/Babylon.js/blob/master/inspector/src/components/actionTabs/tabs/propertyGrids/animations/popupComponent.tsx :)

Reply:
got it. :)